src/main/jbake/content/release-model.ad

```diff
@@ -0,0 +1,255 @@
+////
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+////
+= Making a model release
+:jbake-type: page
+:jbake-tags: building
+:jbake-status: published
+:idprefix:
+
+## Release Preparation
+
+- Elect a release manager.
+- Create a ticket for a new model release in JIRA. If you do not have permission to do so just ask to be given permissions on the mailing list.
+
+## Steps for the Release Manager
+
+The following steps need only to be performed once.
+
+- Make sure you have your PGP key entered into https://id.apache.org/.
+  Your KEYS will then be present in https://people.apache.org/keys/group/opennlp.asc.
```

Comment:
Hmm, that's 404 for me right now, https://people.apache.org/keys/group/opennlp.asc

Reply:
Will check. That is a copy of release.html (so should also be fixed there)

Reply:
Keys are only managed via SVN / CDN via dist/release nowadays, see https://people.apache.org/keys/ - see Issue [OPENNLP-1438](https://issues.apache.org/jira/browse/OPENNLP-1438) for updating `release.ad`.